tools/json_tools/cddatags.py

```diff
@@ -72,7 +72,7 @@ def main(args):
     json_tags_lines = [make_tags_line(*d) for d in definitions]
     existing_tags_lines = []
     try:
-        with open(TAGS_FILE, 'rb') as tags_file:
+        with open(TAGS_FILE, 'rb', encoding="utf-8") as tags_file:
```

Comment:
FYI, you accidentally broke this because you shouldn't add an encoding option when opening in binary mode.  I've PRed a fix as part of some unrelated changes in #49980 where I noticed it.

Reply:
Oops, I don't know how I missed that 😅. Thanks for the fix!